src/lib/actions/gallery/gallery.ts

```
@@ -275,6 +275,129 @@ export async function deleteGallery(id: string) {
+        fileUrl: string
+        name: string
+        description?: string | null
+      } | null = null
+
+      switch (ref.sourceType) {
+        case 'main':
+          const [mainImg] = await db
+            .select()
+            .from(imageData)
+            .where(eq(imageData.id, parseInt(ref.imageId)))
+            .limit(1)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)